scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -16,260 +16,341 @@
 
 set -e
 
-if [ $EUID -ne 0 ]; then
-    echo "ERROR: this script must be run as root to install 3rd party packages." >&2
-    echo "Please try again with \"sudo -E $0\", or as root." >&2
-    exit 1
-fi
+#===================================================================================================
+# Option parsing
 
-params=$@
+all_comp=(opencv_req opencv_opt python dev myriad dlstreamer installer pot)
```

Comment:
> opencv_req opencv_opt

- `opencv opencv_optional`
- or `opencv opencv_extra`

?

Reply:
I like variant with `opencv opencv_opt` (5 letters shorter than `optional` 😃), but I think we can leave it as is to avoid changes in docker_ci PR.